vs_grasping_loop_rate: 5
manual_grasping: true